files/postgres/postgresql.conf

```diff
@@ -0,0 +1,815 @@
+# -----------------------------
```

Comment:
do we want the whole of the file, or just the settings themselves for this?

`grep ^[a-z] postgresql.conf`

Reply:
I decided to do it this way so users had an easy way to modify other settings as necessary.